src/sqlfluff/core/parser/segments/base.py

```diff
@@ -824,39 +853,55 @@ def to_tuple(
         code_only: bool = False,
         show_raw: bool = False,
         include_meta: bool = False,
+        include_position: bool = False,
     ) -> TupleSerialisedSegment:
         """Return a tuple structure from this segment."""
         # works for both base and raw
 
+        # Build the base tuple (2 elements)
+        base_tuple: Union[
+            tuple[str, str],
+            tuple[str, tuple[TupleSerialisedSegment, ...]],
+        ]
         if show_raw and not self.segments:
-            return (self.get_type(), self.raw)
+            base_tuple = (self.get_type(), self.raw)
         elif code_only:
-            return (
+            base_tuple = (
                 self.get_type(),
                 tuple(
                     seg.to_tuple(
                         code_only=code_only,
                         show_raw=show_raw,
                         include_meta=include_meta,
+                        include_position=include_position,
```

Comment:
wha'ts the reasoning behind passing this as an extra argument if it'll always be set as include_meta in the cli input?